ENCODE-DCC/encoded
# Connects to #22  Start up local dev site

Branched off of basic-build.

Some changes necessary to start/use site locally:
--added users for the group to the user.json (allow login)
--comment out elasticsearch for now (elasticsearch/create_mapping.py was generating some errors) preventing the site from starting [commented out in dev_servers.py]
--set gene_name so that it matches in gene.py and gene.json schema file
--remove some donor info that was throwing an error

To test: 
--  `git fetch`  (to fetch remote branches)
-- `git checkout 22_kd_startUpLocalDevSite`
-- `python3.4 bootstrap.py -v 2.3.1 --setuptools-version 15.2`
-- `bin/buildout`
-- terminal 1: `bin/dev-servers development.ini --app-name app --clear --init --load`
-- terminal 2: `bin/pserve development.ini`

You should be able to login in at http://localhost:6543/ using Persona
